Logs left over from previous incarnation break `make deploy`

https://gitlab.anvil.gi.ucsc.edu/ucsc/azul/-/jobs/2176

```
Error: Creating CloudWatch Log Group failed: ResourceAlreadyExistsException: The specified log group already exists:  The CloudWatch Log Group '/aws/apigateway/azul-indexer-anvilbox' already exists.
  on api_gateway.tf.json line 222, in resource[7].aws_cloudwatch_log_group[0].indexer:
 222:                     }
Error: Creating CloudWatch Log Group failed: ResourceAlreadyExistsException: The specified log group already exists:  The CloudWatch Log Group '/aws/apigateway/azul-service-anvilbox' already exists.
  on api_gateway.tf.json line 671, in resource[25].aws_cloudwatch_log_group[0].service:
 671:                     }
Error: error creating Elasticsearch Domain (azul-index-anvilbox): ValidationException: The Resource Access Policy specified for the CloudWatch Logs log group /aws/aes/domains/azul-index-anvilbox/search-logs does not grant sufficient permissions for Amazon OpenSearch Service to create a log stream. Please check the Resource Access Policy.
  on elasticsearch.tf.json line 115, in resource[4].aws_elasticsearch_domain[0].index:
 115:                     }
make[1]: *** [Makefile:48: auto_apply] Error 1
make[1]: Leaving directory '/builds/ucsc/azul/terraform'
make: *** [Makefile:86: auto_deploy] Error 2
```

To reproduce destroy a deployment, bump the incarnation counter then redeploy,

--- a/README.md
+++ b/README.md
@@ -772,10 +772,16 @@ It is possible that this also works when changing `AZUL_PRIVATE_API` from `0` to
 
 ### Troubleshooting
 
+
+#### Transient SQS errors during `make deploy`
+
 Transient errors might be encountered during the deploy such as `SQS Error Code:
 AWS.SimpleQueueService.NonExistentQueue. SQS Error Message: The specified queue
 does not exist for this wsdl version` In such cases rerunning `make deploy`
 should resolve the issue.
+
+
+#### CloudWatch log group `ResourceAlreadyExistsException`
 
 [aws_cloudwatch_log_group]: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudwatch_log_group
 
@@ -793,12 +799,18 @@ exists` is encountered, follow the steps below to import the
 
 5. `make deploy`
 
+
+#### Service account SAM access causes `azul.RequirementError`
+
 If the error `azul.RequirementError: The service account (SA) '...' is not
 authorized to access ... or that resource does not exist. Make sure that it
 exists, that the SA is registered with SAM and has been granted read access to
 the resource` is encountered, ask an administrator of the Terra group `azul-dev`
 to add the service account as specified in the error messaged to that group. See
 [2.3.4 Google Cloud, TDR, and SAM](#234-google-cloud-tdr-and-sam) for details.
+
+
+#### Unresponsive deployment caused by `KMSAccessDeniedException`
 
 [KMSAccessDeniedException]: https://aws.amazon.com/premiumsupport/knowledge-center/lambda-kmsaccessdeniedexception-errors/
 

--- a/README.md
+++ b/README.md
@@ -786,17 +786,17 @@ exists` is encountered, follow the steps below to import the
 1. `cd terraform`
 
 2. `terraform import aws_cloudwatch_log_group.indexer /aws/apigateway/azul-indexer-foo`
- 
+
 3. `terraform import aws_cloudwatch_log_group.service /aws/apigateway/azul-service-foo`
- 
+
 4. `cd ..`
- 
+
 5. `make deploy`
 
 If the error `azul.RequirementError: The service account (SA) '...' is not
-authorized to access ... or that resource does not exist. Make sure that it 
-exists, that the SA is registered with SAM and has been granted read access to 
-the resource` is encountered, ask an administrator of the Terra group `azul-dev` 
+authorized to access ... or that resource does not exist. Make sure that it
+exists, that the SA is registered with SAM and has been granted read access to
+the resource` is encountered, ask an administrator of the Terra group `azul-dev`
 to add the service account as specified in the error messaged to that group. See
 [2.3.4 Google Cloud, TDR, and SAM](#234-google-cloud-tdr-and-sam) for details.
 
@@ -806,7 +806,7 @@ After a successful invocation of `make deploy`, if the deployment is unresponsiv
 and CloudWatch shows logs entries in the `/aws/apigateway/…` log group but not in
 `/aws/lambda/…`, first confirm whether the issue is the known
 [KMSAccessDeniedException] error. In the AWS Console, go to the Lambda function
-details page, click on the `Test` tab, and click on the `Test` buttton. 
+details page, click on the `Test` tab, and click on the `Test` buttton.
 
 Note that it is normal for some Lambda functions to fail the test due to the
 parameters of the test event. Examine the error message to determine if the

